src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {
+        age
+          ? <p className="Person__age">{`I am ${age}`}</p>
+          : undefined
+      }
```

Comment:
nice solution but better to use &&

age &&  `<p` className="Person__age">{`I am ${age}`}</p>

Reply:
+1